content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -58,37 +83,188 @@ helm repo update
 
 ### Configure the Datadog Agent
 
-Use a YAML file to specify the Helm chart parameters for the [Datadog Agent chart][4].
+{{< tabs >}}
+{{% tab "Datadog Operator" %}}
+After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the deployment of the Datadog Agent, Cluster Agent and Cluster Checks Runners (if used) in your Kubernetes cluster. The Datadog Agent will be deployed as a DaemonSet, running a pod on every node of your cluster.
+
+1. Use datadog-agent.yaml file to specify your `DatadogAgent` deployment configuration.
+
+{{< code-block lang="yaml" filename="datadog-agent.yaml" collapsible="true" >}}
+   apiVersion: datadoghq.com/v2alpha1
+   kind: DatadogAgent
+   metadata:
+     name: datadog
+   spec:
+     global:
+       clusterName: <CLUSTER_NAME>
+       site: <DATADOG_SITE>
+       credentials:
+         apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+         appSecret:
+           secretName: datadog-secret
+           keyName: app-key
+{{< /code-block >}}
+
+- Replace `<CLUSTER_NAME>` with a name for your cluster.
+- Replace `<DATADOG_SITE>` with your [Datadog site][1]. Your site is {{< region-param key="dd_site" code="true" >}}. (Ensure the correct SITE is selected on the right).
+
+2. Switch the Datadog Agent image to use builds with embedded OpenTelemetry Collector:
+
+{{< code-block lang="yaml" filename="datadog-agent.yaml" collapsible="true" >}}
+  ...
+  override:
+    # Node Agent configuration
+    nodeAgent:
+      image:
+        name: "gcr.io/datadoghq/agent:7.62.2-ot-beta"
+        pullPolicy: Always
+{{< /code-block >}}
+
+<div class="alert alert-info">This guide uses a Java application example. The <code>-jmx</code> suffix in the image tag enables JMX utilities. For non-Java applications, use <code>7.62.2-ot-beta</code> instead.<br> For more details, see <a href="/containers/guide/autodiscovery-with-jmx/?tab=helm">Autodiscovery and JMX integration guide</a>.</div>
+
+By default, the Agent image is pulled from Google Artifact Registry (`gcr.io/datadoghq`). If Artifact Registry is not accessible in your deployment region, [use another registry][2].
+
+3. Enable the OpenTelemetry Collector and configure the essential ports:
+
+{{< code-block lang="yaml" filename="datadog-agent.yaml" collapsible="true" >}}
+  ...
+  # Enable Features
+  features:
+    otelCollector:
+      enabled: true
+      ports:
+        - containerPort: 4317
```

Comment:
If no ports are specified here then the operator will default to binding `4317/4318` ([see here](https://github.com/DataDog/datadog-operator/blob/a4f0b7c0c0b54589a9b5570e6f06f388dfe5a75d/internal/controller/datadogagent/feature/otelcollector/feature.go#L66-L80)). This option is to be used if you pass non default ports in the otel-config.yaml, in which case you need to respecify the ports here so the operator knows how to bind them. 

I think from this example it appears like setting this is necessary, whereas it's only when using non-default ports.

Reply:
Good point, thanks! Added a note about default port binding and how to override it. 🙇